hw/ip/pinmux/data/pinmux_sec_cm_testplan.hjson

```diff
@@ -0,0 +1,33 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// Security countermeasures testplan extracted from the IP Hjson using reggen.
+//
+// This testplan is auto-generated only the first time it is created. This is
+// because this testplan needs to be hand-editable. It is possible that these
+// testpoints can go out of date if the spec is updated with new
+// countermeasures. When `reggen` is invoked when this testplan already exists,
+// It checks if the list of testpoints is up-to-date and enforces the user to
+// make further manual updates.
+//
+// These countermeasures and their descriptions can be found here:
+// .../pinmux/data/pinmux.hjson
+//
+// It is possible that the testing of some of these countermeasures may already
+// be covered as a testpoint in a different testplan. This duplication is ok -
+// the test would have likely already been developed. We simply map those tests
+// to the testpoints below using the `tests` key.
+//
+// Please ensure that this testplan is imported in:
+// .../pinmux/data/pinmux_testplan.hjson
```

Comment:
Is this file used? Are we only using the corresponding autogen file?

Reply:
This autogenerated testplan is not used (and the file referenced in the line above does not exist either), but we will need to check it in regardless, otherwise it will show up as a untracked file when people run `make -C hw`, which would be annoying.

Reply:
Ack